src/main/java/com/akathist/maven/plugins/launch4j/Launch4jMojo.java

```diff
@@ -490,6 +505,34 @@ private void doExecute() throws MojoExecutionException {
         }
     }
 
+    private void processRequireAdminRights() throws MojoExecutionException {
+        if (requireAdminRights) {
+            getLog().warn("Modifying the resulting exe to always require Admin rights.");
+            getLog().warn("Make sure it's necessary. Consider writing your own manifest file.");
+
+            if (manifest != null) {
+                getLog().warn("manifest param is already set, overriding. Make sure that's what's intended.");
+            }
+
+            try {
+                File manFile = new File(basedir, DEF_REQADMMAN_FILE);
+                byte[] manBytes = FileUtils.readResourceAsBytes(DEF_REQADMMAN_RES);
+
+                FileUtils.writeBytesIfDiff(manFile, manBytes);
```

Comment:
Why not to always overwrite the file?

Reply:
No point in touching it if it's already there and correct